cmd/rebase.go

```diff
@@ -0,0 +1,99 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/vanilla-os/abroot/core"
+	"github.com/vanilla-os/orchid/cmdr"
+)
+
+func NewRebaseCommand() *cmdr.Command {
+	cmd := cmdr.NewCommand(
+		"rebase <name>",
+		abroot.Trans("rebase.long"),
+		abroot.Trans("rebase.short"),
+		rebase,
+	)
+
+	cmd.WithBoolFlag(
+		cmdr.NewBoolFlag(
+			"dry-run",
+			"d",
+			abroot.Trans("rebase.dryRunFlag"),
+			false,
+		))
+
+	cmd.WithBoolFlag(
+		cmdr.NewBoolFlag(
+			"delete-old-system",
+			"",
+			abroot.Trans("rebase.deleteOld"),
+			false,
+		))
+
```

Comment:
It would be nice to have a flag for setting if packages should be removed. Might be useful in scripts.

Reply:
Maybe it needs two flags though, one for keeping and one for removing packages, so that it can still ask the user if none is set.